pylab/website/views.py

```diff
@@ -71,7 +71,26 @@ def about(request):
 
 def event_details(request, year, month, day, slug):
     event = get_object_or_404(Event, starts__year=year, starts__month=month, starts__day=day, slug=slug)
-    return render(request, 'website/event_details.html', {'event': event})
+    attendance = Attendance.objects.filter(event=event)
```

Comment:
This variable should be renamed to ```attendees```. Also ```Event.attendees``` field would eliminate the need to explicitly retrieve from database and pass to template context attendees list, since it would be accessible from already retrieved ```Event``` instance.